engine/src/main/java/org/terasology/math/Rotation.java

```diff
@@ -17,24 +17,27 @@
 
 import com.google.common.base.Objects;
 import com.google.common.collect.ImmutableList;
-
+import gnu.trove.map.TByteByteMap;
 import gnu.trove.map.TByteObjectMap;
+import gnu.trove.map.hash.TByteByteHashMap;
 import gnu.trove.map.hash.TByteObjectHashMap;
+import gnu.trove.procedure.TByteObjectProcedure;
+import org.terasology.math.geom.Quat4f;
 
 import java.util.List;
 
-import org.terasology.math.geom.Quat4f;
-
 /**
  * Rotation provides easy access to 90 degree increments of rotations - intended for block-related rotations.
  * <br><br>
  * Uses the fly weight pattern to cache the 64 combinations of rotation.
+ * Note (Marcin Sciesinski): there are actually only 24 possible combinations, the remaining 40 are just duplicates
  *
  * @author Immortius
  */
 public final class Rotation {
 
     private static final TByteObjectMap<Rotation> ROTATIONS;
+    private static final TByteByteMap DUPLICATE_ROTATIONS;
```

Comment:
Is this needed? Why not just put the same rotation into multiple positions of the main map?